src/services/auth.service.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const API_URL = "http://localhost:3000/api/auth/";
+
+class AuthService {
```

Comment:
@TomekStaszkiewicz can we use classes?

Reply:
Yes, I think that we can for cases like this

Reply:
It will be more readable for people who are not so familiar with js/ts